forte/mrdsrg-spin-adapted/sadsrg.cc

```diff
@@ -82,8 +82,12 @@ SADSRG::~SADSRG() {
 void SADSRG::startup() {
     print_h2("Multireference Driven Similarity Renormalization Group");
 
-    // build Fock and cleanup JK in ForteIntegrals
-    build_fock_from_ints();
+    bool JK_safe = foptions_->get_bool("EMBEDDING_JKFOCK");
+
+    // build fock using ForteIntegrals and clean up JK
+    if (JK_safe) {
+        build_fock_from_ints();
+    }
```

Comment:
You should probably have a function`build_fock_custom()` here instead of `init_fock_emb()` later.

Reply:
Sure!